SmartDeviceLink/private/SDLMenuManager.m

```diff
@@ -554,9 +554,9 @@ - (BOOL)sdl_isDynamicMenuUpdateActive:(SDLDynamicMenuUpdatesMode)dynamicMenuUpda
 - (BOOL)sdl_areAllCellArtworksUploaded:(NSArray<SDLMenuCell *> *)cells {
     for (SDLMenuCell *cell in cells) {
         SDLArtwork *artwork = cell.icon;
-        if (artwork != nil && !artwork.isStaticIcon && self.fileManager != nil && ![self.fileManager hasUploadedFile:(artwork)]) {
+        if (artwork != nil && !artwork.isStaticIcon && ![self.fileManager hasUploadedFile:artwork]) {
```

Comment:
I _think_ that this code will cause a failure in the following situation:

`cells` array: `@[cellWithArt, cellWithoutArt]` passed to this method will return `NO`, because the `cellWithoutArt` returns `NO` after the `artwork != nil` check. This should be fixed by some code like:

```objc
if (artwork == nil) { continue; }
```

Because this method should only check if all the artworks are uploaded, not if they all have artwork **and** that artwork is uploaded, if that makes sense.